bundle/README.md

```diff
@@ -52,6 +52,10 @@ You have access to the OpenShift CLI (oc).
 
 1. When `State: Healthy` appears in the `Status` column, Istio is successfully deployed.
 
+## Selecting the Istio version
+
+The `version` field of the `Istio` custom resource definition defines which version of Istio can be deployed. For a list of available versions, see the [versions.yaml](../versions.yaml) file. This can be set using the `Istio Version` when creating a new `Istio` with the OpenShift Container Platform web console.
```

Comment:
Users can also see the list of versions by running `kubectl explain istio.spec.version`.

Reply:
Thanks, I'll add that too.